omod/src/main/webapp/pages/results.gsp

```diff
@@ -52,6 +67,7 @@
     ];
 
     <% if (results.size() > 0) { %>
+    const PATIENT_COLUMN = 3;
```

Comment:
@shivtej1505 Put a comment on what this variable is used for. I also saw a PROGRAM_COLUMN so do describe what it is used for